### DISCUSSION TITLE
善用GitHub · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#125] [Announcements]
# 善用GitHub · 构建我的被动收入

用GitHub来学习、创作与展示。

https://www.bmpi.dev/self/use-github-better/

**Comment by liugezhou** [comment]
邮件订阅失败。

**Comment by MrTlyer** [comment]
通过netlify.com也可免费构建next.js网站，源码放在Github私有库可绑定自有域名，类似于Vercel，感觉访问速度还可以：
https://net.ntnas.top

**Reply by madawei2699** [reply]
> 通过netlify.com也可免费构建next.js网站，源码放在Github私有库可绑定自有域名，类似于Vercel，感觉访问速度还可以：
> 
> https://net.ntnas.top

Netlify不错，我有些网站也部署在上面，比Vercel好在可以部署组织的项目，Vercel只能免费部署个人账号下的仓库

**Reply by jingkeke** [reply]
老感觉国外的 尤其是 受美国法律约束的服务 要警惕, 说不定哪天就不让用了.

**Reply by nanvon** [reply]
@jingkeke 其实国内的服务更不稳定。

**Comment by huangwb8** [comment]
这个SVG的思维导图如何导出为适应HTML的？我在很多博客上看到过，不过都不太了解怎么构建的。请指教哈！谢谢啦 (～￣▽￣)～

**Reply by a90120411** [reply]
看了一下 dom 的 className ：[markmap](https://github.com/markmap/markmap)

**Comment by aydengen** [comment]
👍
